cuebot/src/main/resources/public/dtd/cjsl-1.9.dtd

```diff
@@ -37,7 +37,7 @@
 <!-- ================================================================= -->
 
 <!ELEMENT layers            (layer+)>
-<!ELEMENT layer             (cmd,range,chunk,cores?,threadable?,memory?,gpu?,tags?,limits?,env*,services?)>
+<!ELEMENT layer             (cmd,range,chunk,cores?,threadable?,memory?,gpu?,timeout?,timeout_llu?,tags?,limits?,env*,services?)>
```

Comment:
Let's make these changes as a new schema version rather than updating an existing one.

Reply:
Also this, along with the new database migration, mean we should increment the minor version number in the top-level `VERSION.in` file.